modules/gapi/cmake/detectOneVPL.cmake

```diff
@@ -0,0 +1,34 @@
+if(WITH_GAPI_ONEVPL)
+    if (NOT ONEVPL_DIR)
+        message("Search oneVPL package by PATH")
+        find_package(VPL QUIET)
+        if (VPL_FOUND)
+                message("oneVPL found")
+                set(HAVE_ONEVPL TRUE)
+        else()
+            message("No oneVPL found, clone it from repository")
+            set(ONEVPL_PROJECT_PREFIX "${OpenCV_BINARY_DIR}/3rdparty/oneVPL")
+            configure_file("${CMAKE_CURRENT_LIST_DIR}/DownloadOneVPL.cmake.in" ${ONEVPL_PROJECT_PREFIX}/oneVPL-download/CMakeLists.txt)
+            execute_process(COMMAND ${CMAKE_COMMAND} -G "${CMAKE_GENERATOR}" .
```

Comment:
This doesn't support cross-compilation at least.

Reply:
I do no nothing about that before... But current oneVPL implementation for GAPI means Windows only
cc @dmatveev  could you confirm or reject it?

@alalek  & @dmatveev  what do you think about wrapped `externalProject_add`  and such non-cross `execute_process` for WINDOWS only (if this kind of check in present in build system) or some kind of DEV_* cmake flag?

Reply:
> I do no nothing about that before... But current oneVPL implementation for GAPI means Windows only
cc @dmatveev could you confirm or reject it?

Our target use-cases are Windows only but VPL is cross-platform (I believe) so we could continue using/testing it on Linux, too.

Reply:
done. removed